src/modules/Auth/features/RecoverPassword/RecoverForm/RecoverForm.schema.ts

```diff
@@ -0,0 +1,27 @@
+import * as yup from 'yup';
+
+import i18n from 'i18n';
+import { ACCOUNT_PASSWORD_MIN_LENGTH } from 'shared/consts';
+
+export const newPasswordSchema = () => {
+  const { t } = i18n;
+  const passwordRequired = t('passwordRequired');
+  const passwordConfirmationRequired = t('passwordConfirmationRequired');
+  const passwordMinLength = t('passwordMinLength', { chars: ACCOUNT_PASSWORD_MIN_LENGTH });
+  const passwordBlankSpaces = t('passwordBlankSpaces');
+  const passwordsMustMatch = t('passwordsMustMatch');
+
+  return yup
+    .object({
+      password: yup
+        .string()
+        .required(passwordRequired)
+        .min(ACCOUNT_PASSWORD_MIN_LENGTH, passwordMinLength)
+        .matches(/^(\S+$)/, passwordBlankSpaces),
+      passwordConfirmation: yup
+        .string()
+        .required(passwordConfirmationRequired)
+        .oneOf([yup.ref('password')], passwordsMustMatch),
```

Comment:
it's rare case, but anyway as end user can be confused the cases when input message should be 'Please confirm your password' instead of 'Your passwords do not match':
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/16320432/5f735519-477c-4abb-b332-5994891b8b5d)
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/16320432/bb04cd97-c4ff-4f2b-ab00-db836f4ad720)

 I would expect to receive the message: 
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/16320432/18cc8ba8-2e78-4e56-b7aa-05624ba7d04d)
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/16320432/8c82e8b1-4174-4450-9ec7-79a2c49ed0c1)
```suggestion
        .oneOf([yup.ref('password'), ''], passwordsMustMatch),
```

Reply:
Good catch Aleksandr. Thanks for the suggestion